pallets/ds-accounts/src/lib.rs

```diff
@@ -34,41 +37,53 @@ pub mod prelude {
 /// Structure, specific for each role
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Clone, Default)]
-pub struct Account<Moment, AccountRole> {
+pub struct Account<Instant, AccountRole, AccountManager> {
     pub roles: AccountRole,
-    pub create_time: Moment,
```

Comment:
it's better to leave Moment  as a template parameter name.
https://github.com/paritytech/substrate/blob/840478ae83d02561dca72b5556807894ef6fb74a/frame/aura/src/lib.rs#L245